### PR TITLE
Migrate images from chapter 01 to 04

### PR DESCRIPTION
**Pull request by Curt-Park**
## Done
- Enhanced readability of math formula from Chapter01 ~ Chapter04 (#147 #149 #150 #152 #154)
- Image migration from chapter01 to 04 (#123)

#### Vim Command for Substitution
```bash
: %s/https:\/\/wikidocs.net\/images\/page\/\d*/{{ site.baseurl }}\/img\/chapter_img\/chapter04/
```

### CODE REVIEW

File: contents/chapter01/_posts/21-01-28-01_02_convex_optimization_problem.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ convex function과 convex set 사이에는 다음과 같은 밀접한 관계가
 $$
 
 
-![](https://wikidocs.net/images/page/17206/epigraph.png)
+![]({{ site.baseurl  }}/img/chapter_img/chapter01/epigraph.png)
```
Comment (RRoundTable) [inline contents/chapter01/_posts/21-01-28-01_02_convex_optimization_problem.md:66]: 저희 이미지 convention 아래와 같은 것으로 아는데 이전에 작업이 안되었나보군요

https://github.com/convex-optimization-for-all/convex-optimization-for-all.github.io/blob/main/contribution/_posts/21-02-03-conventions.md#23-image-convention

Comment (Curt-Park) [reply contents/chapter01/_posts/21-01-28-01_02_convex_optimization_problem.md]: 수정했습니다.

**Comment by enfow** [comment]
I think there are sorting issue.

The posting for 02-02 should come out fater than the positng for 02-03.

The file `21-02-11-02_02_03_Operations_that_preserve_convexity.md`'s order value is 8, but the file `21-02-11-02_02_Some_important_examples.md` has order value 9.

**Comment by enfow** [comment]
I checked this branch on my local and all images work well. Thank you.

**Review by enfow — APPROVED** [review]

**Comment by Curt-Park** [comment]
@enfow 문서 순서가 몇 개 잘못되어 있었네요. 수정했습니다.

스크린샷 2021-06-05 오전 12 51 47

**Pull request merged**